src/repos/WeeklyDebriefRepo.ts

```diff
@@ -47,6 +48,11 @@ const createWeeklyDebriefs = async (): Promise<User[]> => {
   const userList = UserModel.find({}).then((users) =>
     Promise.all(users.map((user) => createWeeklyDebrief(user, creationDate, expDate))),
   );
+  if (process.env.NODE_ENV === 'dev') {
```

Comment:
why should this only happen on dev? add a comment

Reply:
Since on prod notifications will only be sent on the weekly cron job, to test out notifications on dev I added the notification handler call in the createWeeklyDebriefs mutation. This shouldn't be used on prod since this adds an await in an asynchronous function.